helix-core/src/main/java/org/apache/helix/task/JobConfig.java

```diff
@@ -218,10 +211,6 @@ public int getMaxAttemptsPerTask() {
     return _maxAttemptsPerTask;
   }
 
-  public int getMaxForcedReassignmentsPerTask() {
-    return _maxForcedReassignmentsPerTask;
```

Comment:
For back-compatiblility, do not remove this method, instead, mark it as deprecated and comment with the field is not used by Helix anymore.

Add a todo comment to delete this method in next major release.